spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
```

Comment:
I agree that it's a good idea to test disabling of inputs (especially if it's conditional), as well as the contents of those inputs that are populated by some kind of custom code. But making sure that Rails correctly fills `:name` with `orphan.name` just seems unnecessary.